### ISSUE TITLE
Change placeholder of string or identifier field

### ISSUE THREAD

**Issue by globallyunique** [freon4dsl/Freon4dsl#323]
How is the placeholder for a name field changed from the name of the property, e.g., the property is 'name' and the field with no value should show <no name>?

**Comment by joswarmer** [comment]
This could be added this the edit language of course.

**Comment by joswarmer** [comment]
One relatively simple way of solving this is to use a simple properties file like:
```
MyConcept.myProperty = Placeholder for property
```

**Comment by globallyunique** [comment]
Are you saying that there is already such a property file or just that this would be the way to support adding it to the edit language?

I tried to follow the code but due to my early stage of understanding Freon and weak javascript skills, I couldn't see how the default was set. Can you give more info on where in the TextBox the default is currently set.?

**Comment by joswarmer** [comment]
No, such a property files ins't there yet, but should be relatively simple to add.

**Comment by joswarmer** [comment]
The placeholder is a property of the TextBox, so this can be changed when creating the box.  POnly in custom projections, there is no way in the .edit file to do this.
